pkg/k8shandler/fluentd.go

```diff
@@ -142,6 +149,16 @@ func (clusterRequest *ClusterLoggingRequest) reconcileFluentdServiceMonitor() er
 			// ServerName can be e.g. fluentd.openshift-logging.svc
 		},
 	}
+	logMetricExporterEndpoint := monitoringv1.Endpoint{
+		Port:   exporterPortName,
+		Path:   "/metrics",
+		Scheme: "http",
+		TLSConfig: &monitoringv1.TLSConfig{
```

Comment:
Is it valid to configure a CA when scraping with a scheme of http?  Also, the endpoint started in main.go does not load any certs.  There seems like a mismatch here.  For fluentd, we actually mount certs provided by the certgen service.  @alanconway thoughts?

Reply:
Any cert files should be passed as command line argumets. The question of where they come from and what is in them is outside the scope of this program. We should follow the same pattern that is used in fluentd for the prometheus plugin.

Reply:
@alanconway I'm asking specifically these questions:
* is it valid to stand up a port and define certs but also use scheme `http`.  My expectation is it should be `https`
* we should pass certs as a cli arg as you mention if we need them.  I think they can be the same certs fluent uses
* If we need https then the socket needs to be set up for that later in main.go

Reply:
@pmoogi-redhat  we are defining certs here.  Please update "scheme" to be "https"